SetupTearDown.ps1

```diff
@@ -49,21 +177,12 @@ Function CleanupRemoting{
 
 } # CleanupRemoting()
 
-#Main-function

```

Comment:
why the change in execution?  This change loses the catch of the exceptions.

Reply:
Wasn't sure we needed the catch, since the script is ending after the catch anyway, but I've added it back.